src/XRP/shared/json-schema.ts

```diff
@@ -0,0 +1,47 @@
+/* Schema for adding type information to JSON objects. */
+
+interface RequestJson {
+  method: string
```

Comment:
we have to call it method because that's what rippled is expecting.  How about we make an enum if we end up implementing more than 2 JSON RPC methods?